UnityProjectDP/Assets/Scripts/Visualization/ClassDiagram/ClassEditor.cs

```diff
@@ -1,24 +1,31 @@
 ﻿using System;
 using System.Collections.Generic;
-using UnityEngine;
-using TMPro;
+using System.IO;
+using AnimArch.Extensions;
+using AnimArch.Parsing;
+using AnimArch.Visualization.Animating;
 using AnimArch.Visualization.UI;
 using Networking;
 using OALProgramControl;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UIElements;
+using Button = UnityEngine.UI.Button;
 
 namespace AnimArch.Visualization.Diagrams
 {
     public class ClassEditor : Singleton<ClassEditor>
     {
         private int _id;
         private Graph _graph;
-        private bool _active;
+        public bool active;
         private GameObject _node;
         private string _relType;
 
         public AttributePopUp atrPopUp;
         public MethodPopUp mtdPopUp;
         public ClassPopUp classPopUp;
+        public ParameterPopUp parameterPopUp;
```

Comment:
Toto sa sice nepridalo teraz, ale je to pridane od nas.
```suggestion
        public AttributePopUp AttributePopUp;
        public MethodPopUp MethodPopUp;
        public ClassPopUp ClassPopUp;
        public ParameterPopUp ParameterPopUp;
```